### PR TITLE
Add type annotation to celery/utils/term.py

### PR DESCRIPTION
**Pull request by em1le**
Relates to #7394

### CODE REVIEW

File: celery/utils/term.py [hunk 1]
```diff
@@ -71,107 +74,107 @@ def _fold_no_color(self, a, b):
...
+def supports_images() -> bool:
+    return sys.stdin.isatty() and ITERM_PROFILE is not None
 
 
-def _read_as_base64(path):
-    with codecs.open(path, mode='rb') as fh:
+def _read_as_base64(path: str) -> str:
+    with open(path, mode='rb') as fh:
         encoded = base64.b64encode(fh.read())
-        return encoded if isinstance(encoded, str) else encoded.decode('ascii')
```
Comment (Nusnus) [inline celery/utils/term.py:171]: This should not be removed

**Review by Nusnus — CHANGES_REQUESTED** [review]
It appears the PR changes more stuff.
It would be easier to review if you can split the changes 🙏

File: celery/utils/term.py (hunk 1 above)
Comment (em1le) [reply celery/utils/term.py:171]: The return value of b64encode() is bytes thus encoded is bytes. 
If I missed something let me know and i'll set it back

### Sources
[b64encode() return value](https://docs.python.org/3.7/library/base64.html#base64.b64encode)

Comment (em1le) [reply celery/utils/term.py:171]: Awaiting your feedback upon my previous answer and i will split changes accordingly.

Comment (Nusnus) [reply celery/utils/term.py:171]: Good catch and thanks for confirming your change!

Please do split the changes and consider this line as approved @em1le .
Thank you!

**Comment by Nusnus** [comment]
🚀🚀🚀

**Comment by Nusnus** [comment]
@em1le I’ll review your PRs today - please mark with “Ready for review” when you’re done preparing the PRs.

**Comment by Nusnus** [comment]
@em1le please resolve conflicts

**Pull request merged**